runtime/v2/builder.go

```diff
@@ -209,7 +202,11 @@ func AppBuilderWithBranch[T transaction.Tx](branch func(state store.ReaderMap) s
 
 // AppBuilderWithTxValidator sets the tx validator for the app.
 // It overrides all default tx validators defined by modules.
-func AppBuilderWithTxValidator[T transaction.Tx](txValidators func(ctx context.Context, tx T) error) AppBuilderOption[T] {
+func AppBuilderWithTxValidator[T transaction.Tx](
```

Comment:
what's up with those changes, btw? Everything is unwrapped, looks weird 😅

Reply:
I have a save hook in vs code which runs [golines](https://github.com/segmentio/golines) to reformat lines longer than 112 characters, personal preference, I can disable for code which I'm editing not authoring, but I think it's worth consideration for readability.